### PR TITLE
create app

### PR DESCRIPTION
**Pull request by quinn**

### CODE REVIEW

File: pkg/apply/apply.go [hunk 1]
```diff
@@ -10,15 +11,37 @@ import (
...
 	cmd := exec.Command("kubectl", "apply", "-f", "-")
 	stdin, err := cmd.StdinPipe()
-	defer stdin.Close()
 
 	if err != nil {
 		return
 	}
 
 	stdin.Write(bytes)
+	stdin.Close()
```
Comment (jkminneti) [inline pkg/apply/apply.go]: Just curious, why no defer? Does the system exit cause issues or something?

File: pkg/events/filter.go [hunk 2]
```diff
@@ -18,9 +19,9 @@ func filter(e *util.RegistryEvent) (event *pendingRelease) {
 	branch := slicedTag[2]
 
 	if name == "tuber" && branch == "master" {
-		return &pendingRelease { name: name, branch: branch }
-	} else {
-		fmt.Println("Ignoring", name, branch)
+		return &pendingRelease{name: name, branch: branch}
```
Comment (jkminneti) [inline pkg/events/filter.go]: Oh right lol

File: pkg/apply/apply.go (hunk 1 above)
Comment (quinn) [reply pkg/apply/apply.go]: the stream needs to be closed before CombinedOutput can be called.

File: pkg/events/filter.go (hunk 2 above)
Comment (quinn) [reply pkg/events/filter.go]: the linter made me do it

File: pkg/events/filter.go [hunk 3]
```diff
@@ -2,25 +2,34 @@ package events
...
 }
 
-func filter(e *util.RegistryEvent) (event *pendingRelease) {
-	filterRegex := regexp.MustCompile(`us\.gcr\.io\/(.*):(.*)`)
-	slicedTag := filterRegex.FindStringSubmatch(e.Tag)
-	name := slicedTag[1]
-	branch := slicedTag[2]
+func filter(e *util.RegistryEvent) (event pendingRelease, err error) {
+	apps, err := k8s.TuberApps()
+	var matchApp k8s.TuberApp
```
Comment (jkminneti) [inline pkg/events/filter.go]: if this was a pointer, this function _could_ look less like javascript
```go
for _, app := range apps {
	if app.ImageTag == e.Tag {
		matchApp = &app
		break
	}
}

if matchApp == nil {
	fmt.Println("Ignoring", e.Tag)
	return
}
```
just an option

**Pull request merged**